docs/pl-drawing/index.md

```diff
@@ -35,6 +35,12 @@ The system of coordinates of the canvas is located at the top/left corner, as il
 | `tolerance-hint`           | string  | "The expected tolerance is 1/2 square grid." | Hint explaining tolerance used when grading objects.                                                                                                                        |
 | `disregard-extra-elements` | boolean | false                                        | If true, extra elements are ignored if they match the same reference object. Otherwise, multiple matching elements will reduce the awarded points.                          |
 | `hide-answer-panel`        | boolean | true                                         | If true, the correct answer is not displayed in the answer panel.                                                                                                           |
+| `aria-description`         | string  | None                                         | Text that describes the diagram in detail. See the [accessibility section](#accessibility)                                                                                  |
+| `aria-label`               | string  | None                                         | Text that describes the diagram. See the [accessibility section](#accessibility)                                                                                            |
```

Comment:
Nit: here and elsewhere, I think it makes more sense to lead with `aria-label`. It's the most important piece (you can get by without a description, but not without a label), and in general I think of labels as coming first (think of a form input with a label on top and detailed description on the bottom).

Reply:
Changed in [6d0a370](https://github.com/PrairieLearn/PrairieLearn/pull/11867/commits/6d0a370a11e121b8306b7898aef90a21df6bed30).